cicecore/cicedyn/analysis/ice_history.F90

```diff
@@ -1320,6 +1322,11 @@ subroutine init_hist (dt)
              "none", secday*c100, c0,                                &
              ns1, f_shear)
 
+         call define_hist_field(n_vort,"vort","%/day",tstr2D, tcstr, &
+             "strain rate (vorticity)",                                  &
+             "none", secday*c100, c0,                                &
+             ns1, f_vort)
+
```

Comment:
Should we specify at which location `vort` is defined ? I see shear, divu and strength do not mention it , but most other fields do.